Assets/Huawei/Demos/CloudDB/CloudDBDemo.cs

```diff
@@ -330,4 +331,18 @@ void OnOnCloudDBZoneSnapshotException(AGConnectCloudDBException exception)
         HMSCloudDBManager.Instance.MRegister = HMSCloudDBManager.Instance.MCloudDBZone.SubscribeSnapshot(cloudDBZoneQuery, cloudDBZoneQueryPolicy, OnCloudDBZoneSnapshot, OnCloudDBZoneSnapshotException);
     }
 
+    public void ExecuteServerStatusQuery()
+    {
+        Debug.Log(TAG + " ExecuteServerStatusQuery");
+        ITask<ServerStatus> serverStatusTask = HMSCloudDBManager.Instance.MCloudDBZone.ExecuteServerStatusQuery();
+
+        serverStatusTask.AddOnSuccessListener(serverStatus =>
+        {
+            Debug.Log(TAG + " ExecuteServerStatusQuery serverStatus.ServerTimestamp:" + serverStatus.ServerTimestamp);
+        });
+        serverStatusTask.AddOnFailureListener(e =>
+        {
+            Debug.LogError(TAG + " ExecuteServerStatusQuery AddOnFailureListener:"+e);
```

Comment:
As I said above, you can use 'String interpolation'.

Also, you can write `error` instead of `e` for more readability.